### PR TITLE
Add specs for first-class calc

### PR DESCRIPTION
**Pull request by nex3**
See sass/sass#818
[skip dart-sass]

### CODE REVIEW

File: spec/values/calculation/calc/error/known_incompatible/minus.hrx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<===> input.scss
```
Comment (Awjin) [inline spec/values/calculation/calc/error/known_incompatible/minus.hrx:1]: Is this file a duplicate of `spec/values/calculation/calc/error/known_incompatible/operators.hrx` ?

File: spec/values/calculation/calc/simplify.hrx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+a {b: calc(1px + 2px + 1%)}
+
+<===> plus/left/output.css
+a {
+  b: calc(3px + 1%);
+}
+
+<===>
+================================================================================
+<===> plus/right/input.scss
```
Comment (Awjin) [inline spec/values/calculation/calc/simplify.hrx:21]: Might be worth adding a comment here explaining this behavior.

File: spec/values/calculation/max.hrx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+Error: c is not a number.
+  ,
+1 | a {b: max(c)}
+  |       ^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/rest/input.scss
```
Comment (Awjin) [inline spec/values/calculation/max.hrx:33]: Should this be an error?

File: spec/values/calculation/min.hrx [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+Error: c is not a number.
+  ,
+1 | a {b: min(c)}
+  |       ^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/syntax/rest/input.scss
```
Comment (Awjin) [inline spec/values/calculation/min.hrx:33]: Should this be an error?

File: spec/values/calculation/min.hrx [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+a {b: min(1px, 2px, 3%)}
+
+<===> perserved/unit/third/output.css
+a {
+  b: min(1px, 2px, 3%);
+}
+
+<===>
+================================================================================
+<===> preserved/math/first/output.css
```
Comment (Awjin) [inline spec/values/calculation/min.hrx:208]: Here and below: some of these are missing `input.scss` files

**Review by Awjin — CHANGES_REQUESTED** [review]
Overall LGTM, just some minor nits

File: spec/values/calculation/max.hrx (hunk 3 above)
Comment (nex3) [reply spec/values/calculation/max.hrx:33]: No—the rest argument causes this to be parsed as the global `max()` function—but it certainly shouldn't be tested here.

**Review by Awjin — APPROVED** [review]

**Pull request merged**